src/ASM/ASMs1D.C

```diff
@@ -585,7 +585,7 @@ bool ASMs1D::updateRotations (const Vector& displ, bool reInit)
 }
 
 
-void ASMs1D::getBoundaryNodes (int lIndex, IntVec& glbNodes) const
+void ASMs1D::getBoundaryNodes (int lIndex, IntVec& glbNodes, int basis) const
```

Comment:
Ah anonymisering. Gotcha.

-------- Opprinnelig melding --------
Fra: kmokstad <notifications@github.com>
Dato: 03.05.2016 22:07 (GMT+01:00)
Til: OPM/IFEM <IFEM@noreply.github.com>
Ko: Arne Morten Kvarving <Arne.Morten.Kvarving@sintef.no>, Author <author@noreply.github.com>
Emne: Re: [OPM/IFEM] PETSc reboot - general domain decomposition (#7)


In src/ASM/ASMs1D.C<https://github.com/OPM/IFEM/pull/7#discussion_r61946187>:

> @@ -585,7 +585,7 @@ bool ASMs1D::updateRotations (const Vector& displ, bool reInit)
>  }
>
>
> -void ASMs1D::getBoundaryNodes (int lIndex, IntVec& glbNodes) const
> +void ASMs1D::getBoundaryNodes (int lIndex, IntVec& glbNodes, int basis) const


trenger ikke variabelen 'basis' her da vi vel ikke har støtte for mixed for 1D, dvs. det holder med "..., int) const"

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/OPM/IFEM/pull/7/files/c7217e143f115795decd5a0825082a79366a80f1..42876e3e4e0331f01b4f774f65f6bce431496644#r61946187>